src/step_definitions/steps.js

```diff
@@ -1179,6 +1179,10 @@ Then('user can see Created Date as descending sort', async function() {
   await page.assert.attributeEquals({ selector: "//div/span[normalize-space(.)='Created Date']/span", locateStrategy: "xpath" }, "class", "icon sort-icon is-small is-desc");
 })
 
```

Comment:
Should this just be "When user selects Import file menu item" since it always retrieves the same item rather than utilizing the string passed in? As is it is a bit misleading about the function's purpose.

Reply:
Yes, I will update it since it was incorrectly using the single quote.

Reply:
I will remove the quotes.

Reply:
I have updated it to use the value inside the quote.

When('user selects {string} menu item', async function name(args1) {
  await page.click({selector:`//a[@role='menuitem'][normalize-space(.)='${args}']`, locateStrategy:"xpath"});
})